ArcGISRuntimeSDKQt_CppSamples/Maps/DisplayDeviceLocationWithNmeaDataSources/DisplayDeviceLocationWithNmeaDataSources.qml

```diff
@@ -0,0 +1,49 @@
+// [WriteFile Name=DisplayDeviceLocationWithNmeaDataSources, Category=Maps]
+// [Legal]
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 2.2
+import Esri.Samples 1.0
+
+Item {
+
+    // add a mapView component
+    MapView {
+        id: view
+        anchors.fill: parent
+    }
+
+    // Declare the C++ instance which creates the scene etc. and supply the view
+    DisplayDeviceLocationWithNmeaDataSourcesSample {
+        id: model
+        mapView: view
+    }
+
+    Button {
+        id: button
+        anchors.horizontalCenter: parent.horizontalCenter
+        y: 5
```

Comment:
couple comments:
* not advisable to use hardcoded x/y values. In this case, you could use `anchors.margins: 5` to achieve the same effect
* I'd opt for the button to be on the bottom. Having at the top is a little distracting and hard to reach with thumb when on a phone